.github/workflows/partial-tests.yml

```diff
@@ -7,18 +7,17 @@ jobs:
   linter:
     runs-on: ubuntu-latest
     steps:
+      # replace for Ruff in the future
       - uses: actions/checkout@v4
-
       - uses: ricardochaves/python-lint@v1.4.0
         with:
-          use-black: false
-          use-flake8: false
-          use-isort: false
-          use-pycodestyle: false
           use-pylint: false
+          use-isort: false
+          use-mypy: false
+          extra-black-options: "--line-length=120 --diff"
+          extra-pycodestyle-options: "--max-line-length=120  --exclude=tests/gui --ignore=E121,E123,E126,E226,E24,E704,W503,W504,E203"
           extra-mypy-options: "--ignore-missing-imports --implicit-optional --no-namespace-packages --exclude (taipy/templates/|generate_pyi.py|tools) --follow-imports skip"
-
-      - uses: chartboost/ruff-action@v1
+          extra-isort-options: "--line-length=120 --force-grid-wrap=10 --multi-line=VERTICAL_HANGING_INDENT --trailing-comma"
```

Comment:
We don't need those options because we are only using this action for mypy(should be replaced by a action dedicated to mypy soon).